### DISCUSSION TITLE
提问答疑

### DISCUSSION THREAD

**Discussion by bunny-sadako** [datawhalechina/learn-python-the-smart-way-v2#19] [提问答疑] [answered]
这个None是怎么回事？
![1693483694628](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/128591902/534d05aa-e719-4073-89f3-f28f1fe28c1f)

**Comment by anine09** [comment, marked as answer]
Hi @bunny-sadako，我们发现你的问题在课程中有提及且在互联网上有专门的解释，所以请参考[提问规则](https://datawhalechina.github.io/learn-python-the-smart-way-v2/Question/question/)自行探索答案。

**Reply by anine09** [reply]
其实，很多问题还是希望同学们能够自己探索一下，分析问题解决问题也是编程过程中应该获得的重要能力之一，我们可以给大家一些引导，但是后面的路还是希望大家能够自己更稳地走好。最后，这个问题我们的视频里已经有讲到了，希望能仔细地看一看课程视频再提问哦。

**Comment by bunny-sadako** [comment]
第一课第二课我都听了，没有啊，请问在哪里讲的啊



---原始邮件---
发件人: "Epsilon ***@***.***&gt;
发送时间: 2023年8月31日(周四) 晚上8:24
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [datawhalechina/learn-python-the-smart-way-v2] 提问答疑 (Discussion #19)




 
其实，很多问题还是希望同学们能够自己探索一下，不要把提问区当成人肉debug区哦分析问题解决问题也是编程过程中应该获得的重要能力之一，我们可以给大家一些引导，但是后面的路还是希望大家能够自己更稳地走好最后，其实这个问题我们的视频里已经有讲到了，希望能仔细地看一看课程视频再提问哦~

**Reply by anine09** [reply]
同学，请你确保你已经完整听完课程内容，或者查阅了互联网上的资料，**更应该确保的是你阅读过[提问规则](https://datawhalechina.github.io/learn-python-the-smart-way-v2/Question/question/)**。

我会建议你再检查一遍第二课：[【数据类型和操作】 聪明办法学Python第二版 【精准空降到 18:45】 ](https://www.bilibili.com/video/BV1Ju4y1B73m/?share_source=copy_web&vd_source=028de643cc39a11666d3a10bca7af7d9&t=1125)

**如果我们再次确认你没有认真遵循提问规则后，该问题将被拒绝回答。**
